src/siteconfig/models.py

```diff
@@ -186,6 +186,13 @@ class SiteConfig(models.Model):
             Check this if you want to cap marks at 100%. This setting is only relevent if you are using mark percentages."
     )
 
+    simplified_course_registration = models.BooleanField(
+        verbose_name="Simple course registration", default=False,
+        help_text="When students are registering for a course, if simple course registration is enabled, they won't have to select choices for \
+            fields with only one option in the course registration form. If all registration fields only have one option, the form will \
+            automatically submit itself."
```

Comment:
help_text="If this option is enabled, when a student registers for a course they will only have to select fields that have more than one option.  If your deck only has a single course, group, and semester, then the user will automatically enrolled in the course without having to submit a form."

Reply:
help text changed.